tests/aritmeticops.py.uast

```diff
@@ -8,6 +8,10 @@ Module {
 .  .  Line: 1
 .  .  Col: 1
 .  }
+.  Properties: {
```

Comment:
I stand corrected; currently the [https://github.com/bblfsh/sdk/blob/master/protocol/native/objecttonoder.go] ObjectToNoder struct doesn't map the endpositions in any way, so that must be changed [I opened an issue](https://github.com/bblfsh/sdk/issues/153). Still I'll merge this one for the other fixes.